### ISSUE TITLE
Label image has more than two classs

### ISSUE THREAD

**Issue by mitalishah25** [zizhaozhang/unet-tensorflow-keras#13]

**Comment by joe1chief** [comment]
Can I have a look of your files and folds tree? I am stuck here too.
